Flakes error on AppVeyor

Faisl persistently unrelated to actual changes.

    Build started
    git clone -q https://github.com/tox-dev/tox.git C:\projects\tox
    git fetch -q origin +refs/pull/508/merge:
    git checkout -qf FETCH_HEAD
    Running Install scripts
    C:\Python36\python -m pip install tox
    Collecting tox
      Downloading tox-2.7.0-py2.py3-none-any.whl (49kB)
    Requirement already satisfied: virtualenv>=1.11.2; python_version != "3.2" in c:\python36\lib\site-packages (from tox)
    Collecting pluggy<1.0,>=0.3.0 (from tox)
      Downloading pluggy-0.4.0-py2.py3-none-any.whl
    Collecting py>=1.4.17 (from tox)
      Downloading py-1.4.33-py2.py3-none-any.whl (83kB)
    Installing collected packages: pluggy, py, tox
    Successfully installed pluggy-0.4.0 py-1.4.33 tox-2.7.0
    C:\Python36\scripts\tox
    GLOB sdist-make: C:\projects\tox\setup.py
    flakes create: C:\projects\tox\.tox\flakes
    flakes installdeps: pytest-flakes>=0.2, pytest-pep8
    flakes inst: C:\projects\tox\.tox\dist\tox-2.7.1.dev25+g87a972c.zip
    flakes installed: apipkg==1.4,colorama==0.3.9,execnet==1.4.1,pep8==1.7.0,pluggy==0.4.0,py==1.4.33,pyflakes==1.5.0,pytest==3.1.1,pytest-cache==1.0,pytest-flakes==2.0.0,pytest-pep8==1.0.6,tox==2.7.1.dev25+g87a972c,virtualenv==15.1.0
    flakes runtests: PYTHONHASHSEED='980'
    flakes runtests: commands[0] | py.test --flakes -m flakes tox tests
    ============================= test session starts =============================
    platform win32 -- Python 3.6.1, pytest-3.1.1, py-1.4.33, pluggy-0.4.0
    tox comes from: 'c:\\projects\\tox\\.tox\\flakes\\lib\\site-packages\\tox\\__init__.py'
    rootdir: C:\projects\tox, inifile: tox.ini
    plugins: pep8-1.0.6, flakes-2.0.0
    collected 308 items
    tox\__init__.py F
    tox\__main__.py .
    tox\_pytestplugin.py .
    tox\_quickstart.py .
    tox\_verlib.py .
    tox\config.py .
    tox\hookspecs.py .
    tox\interpreters.py .
    tox\result.py .
    tox\session.py .
    tox\venv.py .
    tests\conftest.py .
    tests\test_config.py .
    tests\test_interpreters.py .
    tests\test_quickstart.py .
    tests\test_result.py .
    tests\test_venv.py .
    tests\test_z_cmdline.py .
    ================================== FAILURES ===================================
    __________________ pyflakes-check(ignoring ImportStarUsage) ___________________
    C:\projects\tox\tox\__init__.py:8: UndefinedName
    undefined name 'DistributionNotFound'
    ============================ 290 tests deselected =============================
    ============= 1 failed, 17 passed, 290 deselected in 0.97 seconds =============
    ERROR: InvocationError: 'C:\\projects\\tox\\.tox\\flakes\\Scripts\\py.test.EXE --flakes -m flakes tox tests'
    ___________________________________ summary ___________________________________
    ERROR:   flakes: commands failed
    Command exited with code 1

--- a/tox/__init__.py
+++ b/tox/__init__.py
@@ -1,4 +1,4 @@
-from pkg_resources import get_distribution
+from pkg_resources import get_distribution, DistributionNotFound
 
 from .hookspecs import hookspec, hookimpl  # noqa
 
@@ -7,7 +7,6 @@ try:
     __version__ = _full_version.split('+', 1)[0]
 except DistributionNotFound:
     __version__ = '0.0.0.dev0'
-
 
 
 class exception:

